dashboard/app/models/plc/enrollment_unit_assignment.rb

```diff
@@ -68,6 +68,35 @@ def focus_area_positions
     plc_module_assignments.map{ |a| a.plc_learning_module.stage.position unless a.plc_learning_module.is_required? }.compact
   end
 
+  def summarize_progress
+    summary = []
+
+    categories_for_stage = plc_course_unit.script.stages.map(&:flex_category).uniq
+
+    if plc_course_unit.script.professional_learning_course?
+      Plc::LearningModule::MODULE_TYPES.keep_if { |type| categories_for_stage.include?(type)}.each do |flex_category|
```

Comment:
This line mutates `Plc::LearningModule::MODULE_TYPES`. If this is intentional, please make the CONSTANT a non_constant. If this is not-intentional, please fix. Without investigation, this could be related to the unit test failures cited in the PR discussion.